RACK-Ontology/ontology/SYSTEM.sadl

```diff
@@ -50,3 +56,27 @@ SYSTEM_DEVELOPMENT
 	is a type of ACTIVITY.
 
 	developedBy (note "Organization that is responsible for producing a system") describes SYSTEM_DEVELOPMENT with values of type AGENT.
+
+FUNCTION  
+	(note "defines functionality / capability")
+	is a type of ENTITY.
+
+	// If we introduce "parent" property for THING, then we could drop parentFunction.	
+	parentFunction (note "allows for decomposition of functionality") describes FUNCTION with a single value of type FUNCTION.
+	parentFunction is a type of wasDerivedFrom.
+	
+	// performedBy is inverse of function; so could be dropped.
+	performedBy (note "SYSTEM that is providing the functionality") describes FUNCTION with values of type SYSTEM.
+	performedBy is a type of wasDerivedFrom.
+
+// Later we could add more structure to OP_ENV.
+OP_ENV   
```

Comment:
What things do we anticipate would relate to one of these?

Reply:
We will add more details as we understand more about how other performers will structure assurance cases.